coalib/bears/Bear.py

```diff
@@ -273,16 +276,24 @@ def __init__(self,
                               time limit, use 0.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
+        :param profile:       Enable profiling on this bear. Contains path or
+                              name of directory where profiled files will be
+                              dumped, if ``True`` is provided files will be
+                              saved to the current directory.
         """
         Printer.__init__(self)
 
         if message_queue is not None and not hasattr(message_queue, 'put'):
             raise TypeError('message_queue has to be a Queue or None.')
+        if profile and debugger:
+            raise ValueError('Invalid argument passed, run debugger or '
+                             'profiler one at a time on {}'.format(self.name))
```

Comment:
Would rephrase that:
> Can't run debugger and profiler at the same time.

(btw it's imo not necessary to include the bear name, since you see it in the traceback which bear raised the error)